docs/application/design/TOC.md

```diff
@@ -138,67 +134,20 @@
 ### [Font Style](/application/design/tv/styles/font-style.md)
 ## UX Overview
 ### [Overview](/application/design/tv/ux-overview.md)
-### App Layer
-#### [Overview](/application/design/tv/ux-overview/app-layer.md)
-#### [Content Area](/application/design/tv/ux-overview/app-layer/content-area.md)
-#### [Assist View](/application/design/tv/ux-overview/app-layer/assist-view.md)
-#### [Notification](/application/design/tv/ux-overview/app-layer/notification.md)
-#### [Toast](/application/design/tv/ux-overview/app-layer/toast.md)
-#### [Controls](/application/design/tv/ux-overview/controls.md)
-#### [Hardware Keys](/application/design/tv/ux-overview/hardware-keys.md)
+### [App Layer](/application/design/tv/ux-overview/app-layer.md)
+### [Controls](/application/design/tv/ux-overview/controls.md)
+### [Hardware Keys](/application/design/tv/ux-overview/hardware-keys.md)
 ## Patterns
 ### [Overview](/application/design/tv/patterns.md)
-### App Structure
-#### [Overview](/application/design/tv/patterns/app-structure.md)
-#### [Source Screen](/application/design/tv/patterns/app-structure/source-screen.md)
-#### [Partial Screen](/application/design/tv/patterns/app-structure/partial-screen.md)
-#### [Full Screen](/application/design/tv/patterns/app-structure/full-screen.md)
-#### [Detail Screen](/application/design/tv/patterns/app-structure/detail-screen.md)
-### Screen Layout
-#### [Overview](/application/design/tv/patterns/screen-layout.md)
-#### [Header Area](/application/design/tv/patterns/screen-layout/header-area.md)
-#### [Content Area](/application/design/tv/patterns/screen-layout/content-area.md)
-#### [Footer Area](/application/design/tv/patterns/screen-layout/footer-area.md)
-### Navigation
-#### [Overview](/application/design/tv/patterns/navigation.md)
-#### [Back Navigation](/application/design/tv/patterns/navigation/back-navigation.md)
-#### [Scrolling](/application/design/tv/patterns/navigation/scrolling.md)
-#### [Focusing](/application/design/tv/patterns/navigation/focusing.md)
-#### [Back Key Information](/application/design/tv/patterns/navigation/back-key-information.md)
-### Actions
-#### [Overview](/application/design/tv/patterns/actions.md)
-#### [Cancel or Confirm](/application/design/tv/patterns/actions/cancel-or-confirm.md)
-#### [Delete](/application/design/tv/patterns/actions/delete.md)
-#### [Multi-select](/application/design/tv/patterns/actions/multi-select.md)
-#### [Pin or Unpin](/application/design/tv/patterns/actions/pin-or-unpin.md)
-#### [Notifications](/application/design/tv/patterns/notifications.md)
```

Comment:
Question : Is this `notifications.md` page is discarded?

Reply:
done